gluecodium/src/main/resources/templates/jni/utils/JniBaseHeader.mustache

```diff
@@ -28,7 +28,7 @@
 // JNI_OnLoad to allow for a custom implementation or similar. In that case the user
 // needs to ensure it is called manually. If it has a custom name, declare the function.
 #ifdef GLUECODIUM_JNI_ONLOAD
-jint GLUECODIUM_JNI_ONLOAD( JavaVM* vm, void* );
+JNIEXPORT jint GLUECODIUM_JNI_ONLOAD( JavaVM* vm, void* );
 #else
 # define GLUECODIUM_JNI_ONLOAD JNI_OnLoad
```

Comment:
Perhaps, the export is also needed when non-custom version of the function is defined.